sql-indent.el

```diff
@@ -136,10 +136,11 @@ whitespace, or at the beginning of the buffer."
       (unless (eq (point) (point-min))
 	(forward-char -1))
       (let ((pps (syntax-ppss (point))))
-	(if (nth 4 pps)                 ; inside a comment?
-	    (goto-char (nth 8 pps))     ; move to comment start than repeat
-	    (throw 'done (point))       ; done
-	    )))))
+	(cond
+	 ((or (nth 4 pps) (nth 3 pps))  ; inside a comment or a string ?
```

Comment:
This is not correct, as strings are not syntactic white-space, and this function should not skip over them.  I think this change will break indentation in other cases.

I understand the problem you were trying to fix, but I don't think this is the right solution.